composer.json

```diff
@@ -39,7 +39,6 @@
             "N98": "src"
         },
         "psr-4": {
```

Comment:
Can be removed?

Reply:
Yes, can be removed. Left it in by intention as a reminder to check if the project can't go to PSR-4 and then would reduce diff display. I would check that in a follow-up and leave it like this in this PR. OK?